dmscripts/generate_buyer_email_list.py

```diff
@@ -13,7 +13,7 @@ def inner_function(user):
         try:
             briefs = client.find_briefs(user_id=user['id'])['briefs']
             if briefs:
-                title_statuses = [u"; ".join([u"{} - {}".format(brief['title'], brief['status']) for brief in briefs])]
```

Comment:
We should probably keep these until we switch the script using this to Py3. It's a stylistic change for Python 3, but it can break unicode strings in Python 2 environments.